exercises/practice/grade-school/GradeSchoolTest.php

```diff
@@ -41,26 +19,96 @@ protected function setUp(): void
         $this->school = new School();
     }
 
-    public function testAddStudent(): void
+    /**
+     * uuid: 9337267f-7793-4b90-9b4a-8e3978408824
+     * @testdox Add a student
+     */
+    public function testAddAStudent(): void
+    {
+        $this->school->add('Aimee', 2);
+        $this->assertCount(1, $this->school->grade(2));
+    }
+
+    /**
+     * uuid: 6d0a30e4-1b4e-472e-8e20-c41702125667
+     * @testdox Student is added to the roster
+     */
+    public function testStudentIsAddedToTheRoster(): void
     {
-        $this->school->add('Claire', 2);
-        $this->assertContains('Claire', $this->school->grade(2));
+        $this->school->add('Aimee', 2);
+        $this->assertContains('Aimee', $this->school->grade(2));
```

Comment:
This is the second API breaking change we want to make, as canonical data wants to have it. Rename `studentsByGradeAlphabetical()` to `roster()` and use it here:

```suggestion
        $actual = $this->school->roster();

        $this->assertEqualsCanonicalizing(
            ['Aimee'],
            array_values($actual)
        );
```